apps/web/src/pages/brand/tabs/LayoutsListPage.tsx

```diff
@@ -170,9 +170,12 @@ export function LayoutsListPage({ handleLayoutAnalytics }: LayoutsListPageProps)
               handleLayoutAnalytics('Create new layout btn clicked');
             }}
           >
-            <Text gradient={!readonly} color={colors.B60}>
-              + Create New Layout
-            </Text>
+            <Group spacing={8}>
+              <PlusFilled width={24} height={24} />
+              <Text gradient={!readonly} color={colors.B60} weight="bold">
+                Add New Layout
+              </Text>
+            </Group>
```

Comment:
there should be already a button created for this on the new Integrations List page, can we reuse it and move to the components folder?

Reply:
Created the reusable component